editions/tw5.com/tiddlywiki.info

```diff
@@ -21,6 +21,7 @@
 		"tiddlywiki/readonly"
 	],
 	"languages": [
+    "en-PH"
```

Comment:
You should remove the languages setting from your final PR, since it will be merged to mast. 
Adding it for testing is OK .

Reply:
You can add it to the full-edition https://github.com/TiddlyWiki/TiddlyWiki5/blob/master/editions/full/tiddlywiki.info

Reply:
It makes sense now, I was wondering where the other languages are. `tw5.com/tiddlywiki.info` is for testing, `full/tiddlywiki.info` is for PR. Made the adjustment. Thank you!